rocketpy/motors/solid_motor.py

```diff
@@ -65,8 +65,10 @@ class SolidMotor(Motor):
     SolidMotor.grain_initial_mass : float
         Initial mass of each grain in kg.
     SolidMotor.dry_mass : float
-        The total mass of the motor structure, including chambers
-        and tanks, when it is empty and does not contain any propellant.
+        The total mass of the motor structure, including chambers, bulkheads,
+        screws, and others. This should be taken when the motor is empty and
+        does not contain any propellant. You should not double count a component
+        that is already accounted for in the rocket class.
```

Comment:
This is a better description then what we have at the liquids and hybrids motors _dry_mass_ `__init__` docs. I'd say its a good ideia to have this in those classes as well, just don't forget to add _tanks_ to the list of included structures

Reply:
Agrred.

I just can't do that right now.

Reply:
Solved by [8d0169e](https://github.com/RocketPy-Team/RocketPy/pull/445/commits/8d0169ec64c8228196cd6d14ff883f907ace0045)